<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminInterface</class>
 <widget class="QWidget" name="adminInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理员</string>
  </property>
  <widget class="QPushButton" name="Button_info_inquiry">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>370</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>信息查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_show_balance">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_welcome">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎使用管理员系统！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_courier_manage">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>450</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>快递员管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_assign_parcel">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>快递分配</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_balance">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>余额：</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>831</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_parcel_assign">
    <widget class="QTableWidget" name="tableWidget_parcel_assign">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>511</width>
       <height>481</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_parcel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>16</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>待分配快递：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_select_courier">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>200</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_courier">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>指定快递员：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_assign">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>290</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>分配快递</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_inquiry">
    <widget class="QCheckBox" name="checkBox_user_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_parcel_id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>快递ID</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_time">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>时间</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sender">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>发送人</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_receiver">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>接收人</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_courier">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>快递员</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_user_name">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_parcel_id">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_time">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sender">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_receiver">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_courier">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_by_keyword">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>39</y>
       <width>82</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>条件查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_show_all">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>60</y>
       <width>82</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>显示全部</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_parcel_result">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>查询结果</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_parcel_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>快递信息查询：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_person_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>人员信息查询：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_by_user">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>420</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户查询</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_person_user_name">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>420</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_by_courier">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>450</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>快递员查询</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_person_courier">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>450</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_person_result">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>380</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>查询结果</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_show_person_result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>511</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_inquiry_person_info">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>490</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_show_parcel_result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>121</y>
       <width>801</width>
       <height>211</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="pushButton_reset_input">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>80</y>
       <width>82</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>重置输入</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_courier_manage">
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>新增快递员：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_account">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>姓名/账号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_account_create">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password_create">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_phone">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>电话：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_phone_create">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_create_account">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>390</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_title_cancel">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>删除快递员账号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cancel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>待删除账号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_account_cancel">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>170</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>210</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button_log_out">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
